l10n/en/brands.ftl

```diff
@@ -137,6 +137,7 @@
 -brand-name-android = Android
 -brand-name-ios = iOS
 -brand-name-linux = Linux
+-brand-name-ubuntu = Ubuntu
```

Comment:
This technically was only used in the Linux.ftl file, but this seems the correct place to place variables like this.